Library/Homebrew/compat/dependencies_helpers.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module DependenciesHelpers
+  module Compat
+    @printed_includes_ignores_warning = false
+
+    def argv_includes_ignores(_argv)
+      unless @printed_includes_ignores_warning
+        odeprecated "Homebrew.argv_includes_ignores", "Homebrew.args_includes_ignores"
+        @printed_includes_ignores_warning = true
+      end
+      args_includes_ignores(Homebrew.args)
+    end
```

Comment:
Should be noted that [`beeftornado/rmtree`](https://github.com/beeftornado/homebrew-rmtree) explicitly passes arguments to `argv_includes_ignores`:

https://github.com/beeftornado/homebrew-rmtree/blob/a26a861/cmd/brew-rmtree.rb#L176

(Something to consider in terms of backwards compatibility)

Reply:
The way it is used in `rmtree`, using

```
Homebrew.argv_includes_ignores(["--installed"])
```

doesn't even return anything with the old version.

Reply:
The way it is used in `rmtree`, using

```
Homebrew.argv_includes_ignores(["--installed"])
```

it always returns the same thing.